src/app/app-platform/ContentAppPlatform.cpp

```diff
@@ -755,6 +756,19 @@ CHIP_ERROR ContentAppPlatform::ManageClientAccess(Messaging::ExchangeManager & e
             {
                 // notify content app about this nodeId
                 app->AddClientNode(subjectNodeId);
+
+                auto tempAccountId = CharSpan::fromCharString(rotatingId.c_str());
+                auto setupPIN = CharSpan::fromCharString(std::to_string(passcode).c_str());
+                auto nodeId = MakeOptional(subjectNodeId); // Q: Is subjectNodeId correct NodeId to use?
+
+                // send login command to content app
+                if (!tempAccountId.empty() && !setupPIN.empty()) {
+                    // Q: Should we only do this if (commissioner) passcode is not 0, e.g when user was shown PIN prompt?
+                    auto status = app->GetAccountLoginDelegate()->HandleLogin(tempAccountId, setupPIN, nodeId);
```

Comment:
@chrisdecenzo Would be very helpful to understand what we want to do here.

Reply:
I was thinking that we would add tempAccountId and setupPIN as arguments to the AddClientNode method and then move the HandleLogin call to within AddClientNode. That method is keeping track of all of the clients of the content app so each time one is added, the HandleLogin should be called (and each time one is removed, the HandleLogout would be called).

Reply:
I ended up not adding the logic to AddClientNode itself, just to keep that method as is. But would be simple to refactor if that is preferred.